melissa/actions/general_conversations.py

```diff
@@ -46,5 +48,12 @@ def how_are_you(text):
 def are_you_up(text):
     tts('For you sir, always.')
 
+def i_love_you(text):
+    replies = ['I have a boyfriend.', 'You are looking for love in wrong place.']
```

Comment:
Let's maybe remove the 'I have a boyfriend.' clause, the second one is significantly smarter.